plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQuerySplit.java

```diff
@@ -60,26 +62,27 @@ public BigQuerySplit(
     {
         this.mode = requireNonNull(mode, "mode is null");
         this.streamName = requireNonNull(streamName, "streamName cannot be null");
+        this.tableName = requireNonNull(tableName, "tableName is null");
         this.schemaString = requireNonNull(schemaString, "schemaString cannot be null");
         this.columns = ImmutableList.copyOf(requireNonNull(columns, "columns cannot be null"));
         this.emptyRowsToGenerate = emptyRowsToGenerate;
         this.filter = requireNonNull(filter, "filter is null");
         this.dataSize = requireNonNull(dataSize, "dataSize is null");
     }
 
-    static BigQuerySplit forStream(String streamName, String schemaString, List<BigQueryColumnHandle> columns, OptionalInt dataSize)
+    static BigQuerySplit forStream(String streamName, String tableName, String schemaString, List<BigQueryColumnHandle> columns, OptionalInt dataSize)
```

Comment:
Why the tableName is before schemaString? reorder. Can we use SchemaTableName here instead?

Reply:
schemaString is string of table definition, not schema name. 

This value holds 3 level identifier (e.g. `projects/development-226103/datasets/test/tables/_pbc_95551a80eb2441d7a343153d7efc5887`), so SchemaTableName looks not suitable. Changed to iamResourceName.